google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITSystemTest.java

```diff
@@ -1537,7 +1537,7 @@ public Void updateCallback(Transaction transaction) throws Exception {
       final StatusRuntimeException invalidArgument = (StatusRuntimeException) rootCause;
       final Status status = invalidArgument.getStatus();
       assertThat(status.getCode()).isEqualTo(Code.FAILED_PRECONDITION);
-      assertThat(status.getDescription()).contains("minimum");
+      assertThat(status.getDescription()).isEqualTo("The requested snapshot version is too old.");
```

Comment:
Question to reviewer: I noticed we've updated this assertion before - is it worth keeping (is the previous assertion enough?)

Reply:
I think we can just delete this line, checking for exact messages does not bring much value and is easy to break.

Reply:
Thanks @wu-hui, I've removed the assertion.